mixin/config.libsonnet

```diff
@@ -0,0 +1,13 @@
+{
+  _config+:: {
+    dashboardTags: ['snowflake-mixin'],
+    dashboardPeriod: 'now-8h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '30m',
+
+    // for alerts
+    alertsWarningLoginFailures: '30',  // %
+    alertsComputeCreditUsageLimit: '5',  // credits/hr
+    alertsServiceCreditUsageLimit: '1',  // credits/hr
```

Comment:
I wonder how common these credit usage limits would be for orgs of varying sizes 🤔

In the future, this is definitely something we want to be configurable within the integration setup/settings